jscomp/test/record_regression.js

```diff
@@ -80,6 +80,16 @@ var h = newrecord$2;
 
 var h10 = newrecord$3;
 
+var foo1 = /* Foo */{
+  name: "foo",
+  age: undefined
```

Comment:
One thing left is here: not print unused field `age`
```js
var foo1 = /* Foo */{
  name: "foo"
}
```

Reply:
Fixed https://github.com/rescript-lang/rescript-compiler/pull/5827/commits/31432007dd9022f6c6b08be2ca82017c3bf70292